packages/docs/src/core/src/spec/transpilation-details.rst

```diff
@@ -96,13 +105,24 @@ To replace Call-type opcodes, we have to pass an existing slice of ``calldata``
       - 2
     * - ``DELEGATECALL``
       - 2
+>>>>>>> newrepo/master
```

Comment:
This looks like an unresolved conflict. Choose one or the other and get rid of the metadata (<<<<<<< HEAD, ========, >>>>>>>).